js/bybit.js

```diff
@@ -1034,6 +1034,8 @@ module.exports = class bybit extends Exchange {
             },
             'precisionMode': TICK_SIZE,
             'options': {
+                'enableUnifiedMargin': undefined,
```

Comment:
Should we use boolean instead of `undefined`?

Reply:
@sc0Vu I don't think so, because `false` means the account is not UAT or unified margin and we want to represent the 'undefined'/unknown status